project/chrome-extension/script-two.js

```diff
@@ -0,0 +1,21 @@
+//Calls the showInfo method after the whole document loads.
```

Comment:
Why does this need to be its own file? Why can it not be added to script.js? 

If this file is required, name it something more descriptive.

Reply:
This was done in the UpdateScript-Two branch.

Reply:
What was done in the UpdateScript-Two branch? I was referring to the name of the script.

Reply:
oh, the raname. Since this created the file, it is most appropriate to make the change here. You will need to make sure that the UpdateScript-Two branch is up to date by doing a pull after the change.

Reply:
The name of the scripts were updated  to be more descriptive in that other branch.

Reply:
> oh, the raname. Since this created the file, it is most appropriate to make the change here. You will need to make sure that the UpdateScript-Two branch is up to date by doing a pull after the change.

Oh, I see; was not aware of that. Noted, thank you.

Reply:
So, doesn't it work if I merge this branch and then the other branch?